website/versions.json

```diff
@@ -1,4 +1,6 @@
 [
+  "v9.3.0",
+  "9.3.0",
```

Comment:
I think we need to remove line 3. Looks like you re-added with the "v" 😄

Reply:
Thanks!  I initially used 9.3.0, and then changed it to v9.3.0. It's fixed now.